<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroProfessor</class>
 <widget class="QDialog" name="CadastroProfessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>363</width>
     <height>169</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nomeLabel">
        <property name="text">
         <string>Nome</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nomeLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="emailLabel">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="emailLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="telefoneLabel">
        <property name="text">
         <string>Telefone</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="telefoneLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="botaoCancelaProfessor">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoCadastraProfessor">
        <property name="text">
         <string>Cadastrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoSelecionaHorariosProfessor">
        <property name="text">
         <string>Selecionar 
 Hor√°rios</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
